src/function/utility/error.cpp

```diff
@@ -0,0 +1,26 @@
+#include "common/exception/runtime.h"
+#include "function/scalar_function.h"
+#include "function/utility/scalar_utility_functions.h"
+
+using namespace kuzu::common;
+
+namespace kuzu {
+namespace function {
+
+struct Error {
+    static void operation(ku_string_t& input, int32_t& result) {
+        result = 0;
+        throw RuntimeException(input.getAsString());
+    }
+};
+
+function_set ErrorFunction::getFunctionSet() {
+    function_set functionSet;
+    functionSet.push_back(
+        std::make_unique<ScalarFunction>(name, std::vector<LogicalTypeID>{LogicalTypeID::STRING},
+            LogicalTypeID::INT32, ScalarFunction::UnaryExecFunction<ku_string_t, int32_t, Error>));
```

Comment:
Can you add a comment about the dummy resultType? I think int32_t is simply a dummy type right?